core/server/services/routing/helpers/fetch-data.js

```diff
@@ -28,6 +28,13 @@ const defaultQueryOptions = {
     }
 };
 
+const defaultDataQueryOptions = {
```

Comment:
@kirrg001 changed the config to be more explicit. Think it's pretty readable this way

Reply:
Looks okay as a hot fix, but IMO the includes shouldn't be here at all. They should be part of the target api version config (long-term).